conf/deployment/vsphere/ipi_1az_rhcos_multus_vsan_3m_3w.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Can we also add one more config for UPI please ?

Reply:
[One already exists](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/vsphere/upi_1az_rhcos_multus_vsan_3m_3w.yaml)